### PR TITLE
Add my old proofread text to Section 1

### PR DESCRIPTION
**Pull request by JesseTG**
The text comes from [here](https://github.com/JesseTG/glm/blob/ef3ee873b51e9e309bf141b4fa987995c5cd7448/doc/glm.tex#L125-L206).

### CODE REVIEW

File: manual.md [hunk 1]
```diff
@@ -208,12 +207,9 @@ glm::mat4 transform(glm::vec2 const& Orientation, glm::vec3 const& Translate, gl
...
 
-There is no dependence with external libraries or external headers such as gl.h, [*glcorearb.h*](http://www.opengl.org/registry/api/GL/glcorearb.h), gl3.h, glu.h or windows.h. However, if &lt;boost/static\_assert.hpp&gt;
-is included, [*Boost static assert*](http://www.boost.org/doc/libs/1_52_0/doc/html/boost_staticassert.html) will be used all over GLM code to provide compiled time errors unless
-GLM is built with a C++ 11 compiler in which case [static\_assert](http://en.cppreference.com/w/cpp/language/static_assert).
-If neither are detected, GLM will rely on its own implementation of static assert.
+GLM does not depend on external libraries or external headers such as `<gl.h>`, [`<glcorearb.h>`](http://www.opengl.org/registry/api/GL/glcorearb.h), `<gl3.h>`, `<glu.h>`, or `<windows.h>`. However, if `<boost/static_assert.hpp>` is included, then [`Boost.StaticAssert`](http://www.boost.org/doc/libs/release/libs/static_assert) will be used to provide compile-time errors.  Otherwise, if using a C++11 compiler, the standard `static_assert` will be used instead. If neither is available, GLM will use its own implementation of `static_assert`.
```
Comment (Groovounet) [inline manual.md]: If you want to use the proper include path, then `<gl.h>, <glcorearb.h>, <gl3.h> and <glu.h>`
are typically not valid. (unless specific compiler argument to make sur path valid.)
The path should be: `<GL/gl.h>, <GLES3/gl3.h>, <GL/glcorearb.h> and <GL/glu.h>`.

File: manual.md [hunk 2]
```diff
@@ -7,10 +7,10 @@
 ---
 ## Table of Contents
 + [0. Licenses](#section0)
-+ [1. Getting started](#section1)
++ [1. Getting Started](#section1)
```
Comment (Groovounet) [inline manual.md]: If you change the style of the title, probably better to do it in a separate PR because this change brings an inconsistency to the manual after the merge.

Comment (JesseTG) [inline manual.md]: Okay, I'll undo it.

File: manual.md (hunk 1 above)
Comment (JesseTG) [inline manual.md]: Oops, my bad.  Will fix.

**Comment by JesseTG** [comment]
Which of these sentences would you prefer, then?

1. "**You** can add sprinkles to ice cream to make it more delicious."
2. "**We** can add sprinkles to ice cream to make it more delicious."
3. "**Add** sprinkles to ice cream to make it more delicious."
4. "**The programmer** can add sprinkles to ice cream to make it more delicious."
5. "**The user** can add sprinkles to ice cream to make it more delicious."

**Comment by Groovounet** [comment]
I would say in order: 2 and 4

"Sprinkles can be added to make ice cream more delicious." 

Is good to me too.

**Comment by JesseTG** [comment]
You got it.

**Comment by Groovounet** [comment]
Thanks!

**Comment by JesseTG** [comment]
@Groovounet How about this?

**Comment by Groovounet** [comment]
This looks great, thanks for contributing!
Christophe

**Comment by JesseTG** [comment]
No problem.  Section 2 coming up soon.
